docs/core/entity/media-player.md

```diff
@@ -236,9 +260,12 @@ Optional. What type of media device is this. It will possibly map to google devi
 
 | Value | Description
 | ----- | -----------
-| tv | Device is a television type device.
+| display | Device is a television or projector type device.
```

Comment:
It is `tv`, not `display`.

Reply:
What you say literally follows the actual code - I get it now that this PR wasn't the right way to open architectural change for a discussion.

Reply:
> What you say literally follows the actual code

Yes, there is nothing else.